### ISSUE TITLE
GSS_data/Migration/ONS-LTIM-Passenger-Survey-4.02 #98 failed

### ISSUE THREAD

**Issue by ajtucker** [ONS-OpenData/ONS-LTIM-Passenger-Survey-4.02#3]
Build 'GSS_data/Migration/ONS-LTIM-Passenger-Survey-4.02' is failing!

Last 50 lines of build output:

```
[...truncated 164 lines...]
[0;32m--> 116[0;31m                 [0mscrape[0m[0;34m([0m[0mself[0m[0;34m,[0m [0mtree[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    117[0m                 [0mscraped[0m [0;34m=[0m [0;32mTrue[0m[0;34m[0m[0;34m[0m[0m
[1;32m    118[0m                 [0;32mbreak[0m[0;34m[0m[0;34m[0m[0m

[0;32m~usr/local/lib/python3.7/site-packages/gssutils/scrapers/ons.py[0m in [0;36mscrape[0;34m(scraper, tree)[0m
[1;32m     14[0m         "//h1/text()")[0].strip()
[1;32m     15[0m     scraper.dataset.issued = parse(tree.xpath(
[0;32m---> 16[0;31m         "//span[starts-with(text(),'Release date:')]/parent::node()/text()")[1].strip()).date()
[0m[1;32m     17[0m     user_requested = tree.xpath(
[1;32m     18[0m         [0;34m"//h2[starts-with(text(),'Summary of request')]"[0m[0;34m[0m[0;34m[0m[0m

[0;31mIndexError[0m: list index out of range
IndexError: list index out of range

[Pipeline] }
[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // script
[Pipeline] }
$ docker stop --time=1 a4fb1c0017918b182a5fc9bba0bb2e9b9111aea5adf2371a103fe1d3d899dce0
$ docker rm -f a4fb1c0017918b182a5fc9bba0bb2e9b9111aea5adf2371a103fe1d3d899dce0
[Pipeline] // withDockerContainer
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Validate CSV)
Stage "Validate CSV" skipped due to earlier failure(s)
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Upload Tidy Data)
Stage "Upload Tidy Data" skipped due to earlier failure(s)
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Test draft dataset)
Stage "Test draft dataset" skipped due to earlier failure(s)
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Publish)
Stage "Publish" skipped due to earlier failure(s)
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Declarative: Post Actions)
[Pipeline] script
[Pipeline] {
[Pipeline] step

```

Changes since last successful build:
No changes

[View full output](https://ci.floop.org.uk/job/GSS_data/job/Migration/job/ONS-LTIM-Passenger-Survey-4.02/98/)

**Comment by ajtucker** [comment]
ONS Scraper issue, should be fixed by GSS-Cogs/gss-utils#2.

**Comment by ajtucker** [comment]
Build was fixed!
